java/org/openobservatory/measurement_kit/android/DnsUtils.java

```diff
@@ -0,0 +1,59 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+package org.openobservatory.measurement_kit.android;
+
+import android.util.Log;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+
+public class DnsUtils {
```

Comment:
@lorenzoPrimi, this branch also factors common functionality that now are implemented directly by netprobe-android and that I needed to have also in the showcase-for-ndt app I am working on